pkg/loghttp/params.go

```diff
@@ -75,6 +75,10 @@ func interval(r *http.Request) (time.Duration, error) {
 	return parseSecondsOrDuration(value)
 }
 
+func hint(r *http.Request) string {
+	return r.Form.Get("hint")
```

Comment:
We should be careful here about using unsanitized user input; this is a common attack vector.

Reply:
Good point. Now we allow only alpha-numeric and `-` characters in the `hint` string!

Reply:
Thanks!